packages/web/src/components/track/mobile/PlaylistTile.tsx

```diff
@@ -295,3 +368,6 @@ const PlaylistTile = (props: PlaylistTileProps & ExtraProps) => {
 }
 
 export default PlaylistTile
+function dispatch(arg0: any) {
```

Comment:
remove? or i guess useDispatch maybe not being used?

Reply:
omg lol. thanks a lot vscode